examples/multiformat_example.html

```diff
@@ -0,0 +1,214 @@
+<!-- -*- Mode: Web; -*- -->
```

Comment:
Why aren't we using the code from http://mlane.devnxs.net/multiformat/?

Reply:
This was added in error as a placeholder.  Updated in 639aae0 to use Lane's example code

Note: There are 2 modifications from Lane's example code to (1) use Prebid v1.2.0 from the CDN and (2) s/appnexusAst/appnexus/g